external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MsSqlServerIntegrationSuite.scala

```diff
@@ -202,4 +205,32 @@ class MsSqlServerIntegrationSuite extends DockerJDBCIntegrationSuite {
     df2.write.jdbc(jdbcUrl, "datescopy", new Properties)
     df3.write.jdbc(jdbcUrl, "stringscopy", new Properties)
   }
+
+  def create_df() : DataFrame = {
+    val tableSchema = StructType(Seq(
+      StructField("serialNum",ByteType,true)
+    ))
+
+    val tableData = Seq (
+      Row(10)
+    )
+
+    spark.createDataFrame(spark.sparkContext.parallelize(tableData),tableSchema)
+  }
+
+  test("SPARK-28151 Test write table with BYTETYPE") {
+    val df1 = create_df()
+    df1.write
+      .format("jdbc")
+      .mode("overwrite")
+      .option("url",jdbcUrl)
+      .option("dbtable","testTable")
```

Comment:
You need spaces after commas

Reply:
fixed. Sorry about this. I realize dev/stylecheck should be run manually for test cases. I presumed its get run automatically when i build. Apparently not for test code.